amd_openvx/openvx/ago/ago_util.cpp

```diff
@@ -1362,17 +1507,23 @@ int agoGetDataFromDescription(AgoContext * acontext, AgoGraph * agraph, AgoData
 		data->ref.type = VX_TYPE_THRESHOLD;
 		const char *s = strstr(desc, ","); if (!s) return -1;
 		char thresh_type[64], data_type[64];
+		uint32_t input_format, output_format;
```

Comment:
@hansely & @LakshmiKumar23 - this section of the code is buggy, try runvx canny to test.

Reply:
@kiritigowda Found the issue. Emailed the details